ui/component/or-map/src/index.ts

```diff
@@ -518,6 +519,14 @@ export class OrMap extends LitElement {
                     this._processNewMarkers(info.addedNodes);
                     this._processRemovedMarkers(info.removedNodes);
                 });
+                this._resizeObserver?.disconnect();
```

Comment:
Could be a risk of a memory leak here; the observer should be cleaned up in the `disconnectedCallback()` not sure we should rely on the browser to do this automatically:

https://stackoverflow.com/questions/67581868/should-i-call-resizeobserver-unobserve-for-removed-elements

@MartinaeyNL I know you've used `ResizeObserver` in a few places, please check if this is relevant there also.

Reply:
Yeah it wasn't really mentioned in the official docs. Indeed a good practice to disconnect anyway in `disconnectedCallback()`, just to prevent possible issues in the future. I'll include it in some insights fixes soon